app/Photo.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Photo extends Model
+{
+    const DATE_FORMAT_IMAGE_NAME = "Y-m-d-G-i-s";
+    const DATE_FORMAT_SORTING_BY_DATE = "Y/m";
+
+    public static function getCountRows()
+    {
+        return self::get()->count();
+    }
+
+    public static function getAllRows()
+    {
+        return self::get();
+    }
+
+    public static function getLastRecord()
+    {
+        return self::orderBy('id', 'desc')->first();
+    }
+
+    private static function saveImageFromRaspi(\Illuminate\Http\Request $request)
+    {
+        $destinationPath = env('RASPI_PHOTO_FOLDER_PATH') . date(self::DATE_FORMAT_SORTING_BY_DATE);
+        $file = $request->file('photo');
+        $filename = date(self::DATE_FORMAT_IMAGE_NAME) . "." . $file->getClientOriginalExtension();
+        $file = $request->file('photo')->move($destinationPath, $filename);
+        return $file;
+    }
+
+    private static function saveInfoToDatabase(string $pathInfo, string $baseName, \Illuminate\Http\Request $request)
+    {
+        $photo = new Photo;
+        $photo->name = "public/" . $pathInfo . "/" . $baseName;
```

Comment:
please convert public to const, use directory separator const

Reply:
created const